content/github/writing-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -215,7 +215,7 @@ You can add emoji to your writing by typing `:EMOJICODE:`.
 
 Typing `:` will bring up a list of suggested emoji. The list will filter as you type, so once you find the emoji you're looking for, press **Tab** or **Enter** to complete the highlighted result.
 
-For a full list of available emoji and codes, check out [emoji-cheat-sheet.com](http://emoji-cheat-sheet.com).
```

Comment:
emojicode

Reply:
I would appreciate if you would not constantly make approves and also only submit suggestions/change requests that do make sense here.